src/test/java/org/opensearch/security/auditlog/compliance/ComplianceAuditlogTest.java

```diff
@@ -306,73 +305,29 @@ public void testUpdate() throws Exception {
             .actionGet();
         }
 
-        TestAuditlogImpl.clear();
-
-        String body = "{\"doc\": {\"Age\":123}}";
-
-        HttpResponse response =  rh.executePostRequest("humanresources/_doc/100?pretty", body, encodeBasicHeader("admin", "admin"));
-        Assert.assertEquals(HttpStatus.SC_CREATED, response.getStatusCode());
-
-        body = "{\"doc\": {\"Age\":456}}";
+        final MessagesNotFoundException ex1 = assertThrows(MessagesNotFoundException.class, () -> {
+            TestAuditlogImpl.doThenWaitForMessage(() -> {
+                final String body = "{\"doc\": {\"Age\":123}}";
+                final HttpResponse response =  rh.executePostRequest("humanresources/_doc/100?pretty", body, encodeBasicHeader("admin", "admin"));
+                Assert.assertEquals(HttpStatus.SC_CREATED, response.getStatusCode());
+            });
+        });
+        assertThat(ex1.getMissingCount(), equalTo(1));
+
+
+        final MessagesNotFoundException ex2 = assertThrows(MessagesNotFoundException.class, () -> {
+            TestAuditlogImpl.doThenWaitForMessage(() -> {
+                final String body = "{\"doc\": {\"Age\":456}}";
+                final HttpResponse response =  rh.executePostRequest("humanresources/_update/100?pretty", body, encodeBasicHeader("admin", "admin"));
+                Assert.assertEquals(HttpStatus.SC_OK, response.getStatusCode());
+            });
+        });
+        assertThat(ex2.getMissingCount(), equalTo(1));
 
-        response = rh.executePostRequest("humanresources/_update/100?pretty", body, encodeBasicHeader("admin", "admin"));
-        Assert.assertEquals(HttpStatus.SC_OK, response.getStatusCode());
-        System.out.println(response.getBody());
-        Thread.sleep(1500);
         Assert.assertTrue(TestAuditlogImpl.messages.isEmpty());
         Assert.assertTrue(validateMsgs(TestAuditlogImpl.messages));
     }
 
-    @Test
-    public void testUpdatePerf() throws Exception {
```

Comment:
This test has been removed because it doesn't verify any behavior of audit logs, all of the functionality has been commented out.  Even with the commented out code added back in, that code does not include any verification or 'performance comparisons'.

Implementing perf tests is difficult to do well - I do not recommend that we invest or build on perf testing scenarios.